src/utility.ts

```diff
@@ -1,3 +1,6 @@
+declare global {
+  var timeStamp: string;
+}
```

Comment:
What was the reason for this? I think we could remove it though as we aren't using timestamps anymore

Reply:
this is for getting the name of stored images, from global variable, because we are not returning filename right?

Reply:
But you never use the global time stamp variable? It’s also a global which we should avoid unless absolutely necessary

Reply:
no i do use it, in the tests, for file existence verification

Reply:
I think we should return file name from the screenshot method then, which will aid in unit tests and we can then removal the global stuff